src/styles/blocks/header.scss

```diff
@@ -0,0 +1,70 @@
+.header {
+  max-width: 1200px;
+  left: 0;
+  top: 0;
+  margin: 0 auto;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: rgb(255, 255, 255);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+
+}
+
+.nav {
+  &__list {
+    padding: 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  &__item {
+    display: flex;
+    padding: 0 10px;
+
+    &:first-child {
+      padding-left: 0;
+    }
+
+    &:last-child {
+      padding-right: 0;
+    }
```

Comment:
Why do you use flex here? It does nothing in this case.
It's logically better to use margin-right here except for the last item.

Reply:
I have used flex for style decoration reset for the list, I agree it's hardcode so I will replace it with list-style: none ) Paddings, just forgot to replace it with margin.  Thanks.